README.md

```diff
@@ -1,12 +1,45 @@
 # discord.py-welcome-bot
 
-A simple bot that can be used to welcome new users to a discord server, sending them a custom private message and adding them to a role.
+A simple bot that can be used to welcome new users to a discord server, send them a custom private message, send a welcome message to a specific channel, and adding them to a role.
```

Comment:
I clearly typed the original in a rush 🤦 `and adding them to a role.` should be `and give them a specific role`